package.json

```diff
@@ -8,8 +8,8 @@
     "bootstrap": "run-s bootstrap:sponsors build",
     "bootstrap:sponsors": "node ./scripts/download-sponsors.js",
     "build": "run-s build:codemirror6 build:repl build:docusaurus build:redirects",
-    "build:docusaurus": "cd website && yarn && yarn build && mv build babel && mkdir build && mv babel build/babel",
```

Comment:
`mv` does not exist for Windows, and there is an [--out-dir](https://docusaurus.io/docs/cli#docusaurus-build-sitedir) option for Docusaurus, so we don't have to move files around manually